README.md

```diff
@@ -25,7 +25,7 @@ An [*anaconda*](https://www.anaconda.com/distribution/) environment is recommend
 There are 5 main methods called in the main.py file. We will refer to them according to the numbers to indicate how they should be commented on at a specific stage of the experiment. 
 
 
-The path value should be set to 'DATASET_DATA_DIR' as shown in [file](https://github.com/CarolRameder/NaroNet/blob/main/NaroNet-main/src/main.py).
+The path value should be set to 'DATASET_DATA_DIR' as shown in line 83 from [file](https://github.com/CarolRameder/NaroNet/blob/main/NaroNet-main/src/main.py).
```

Comment:
Why do you call it DATASET_DATA_DIR path if there's not variable named like that? It's a bit confusing.
Can you also please add the code snippet using the `.
```python
path = '/home/carol/NaroNet-main/NaroNet-main/Endometrial_POLE/'    
```